nextflow.config

```diff
@@ -284,11 +270,26 @@ manifest {
     homePage        = 'https://github.com/genomic-medicine-sweden/tomte'
     description     = """Pipeline to analyse RNAseq from raredisease patients"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
+    nextflowVersion = '!>=24.04.2'
     version         = '2.2.1'
     doi             = '10.5281/zenodo.10828946'
 }
 
+// Nextflow plugins
+plugins {
+    id 'nf-schema@2.1.1' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+}
+
+validation {
+    defaultIgnoreParams = ["genomes"]
+    help {
+        enabled = true
+        command = "nextflow run $manifest.name -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
+        fullParameter = "help_full"
+        showHiddenParameter = "show_hidden"
+    }
```

Comment:
Hm, do you need a `summary { ... }` here? e.g. https://github.com/genomic-medicine-sweden/nallo/blob/ff30f834e2a4b21fb8687ee7db8043cf8629dfba/nextflow.config

Reply:
Done!